src/node_options.cc

```diff
@@ -918,6 +918,11 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             &EnvironmentOptions::test_global_setup_path,
             kAllowedInEnvvar,
             OptionNamespaces::kTestRunnerNamespace);
+  AddOption("--test-rerun",
```

Comment:
How about a more descriptive flag name, `--rerun-failures`?
I think the current name might be misleading

Reply:
do you mean `--test-rerun-failures`?